tron/utils/timeutils.py

```diff
@@ -59,6 +62,8 @@ def duration(start_time, end_time=None):
     if not start_time:
         return None
     last_time = end_time if end_time else current_time()
+    if last_time.tzinfo is None and start_time.tzinfo is not None:
```

Comment:
What about the other cases here, where a user has added a tz or removed one recently?
Won't this results in another naive/aware clash?

Reply:
Yes, you're right. I'm not sure how to handle this case in general if one has a tzinfo and the other doesn't. I was mostly thinking about the case where start does, and end_time=None so we get a naive time from current_time().

Now I realize that this wouldn't give you the right answer if start_time.tzinfo doesn't match the local timezone.

Good news is that we don't actually need this. duration is only used on naive datetimes now.
start time is set here:
https://github.com/Yelp/Tron/blob/4c7e637058f5e371053878fa7a1732cde6732dbb/tron/core/actionrun.py#L234
and end_time here: 
https://github.com/Yelp/Tron/blob/4c7e637058f5e371053878fa7a1732cde6732dbb/tron/core/actionrun.py#L305

What if I just take out this "fix" and add a warning that you shouldn't mix timezones?

Reply:
A warning sounds good